src/commissaire/models/__init__.py

```diff
@@ -159,6 +159,20 @@ def to_json(self, secure=False):
             self._struct_for_json(secure=secure),
             default=lambda o: o._struct_for_json(secure=secure))
 
+    def to_dict(self, secure=False):
+        """
+        Returns a dict representation of this model. This is different than
+        using __dict__ as the returned data will be model specific only.
+
+        :param secure: If the structure needs to respect _hidden_attributes.
```

Comment:
The way this is worded, here and in `to_json`, often confuses me and I have to go read the logic.

How about: `:param secure: Include _hidden attributes in the return value.`